libs/plugin/src/generators/convert-di-to-inject/generator.ts

```diff
@@ -201,33 +219,28 @@ export async function convertDiToInjectGenerator(
 								isReadonly || options.includeReadonlyByDefault || false,
 							leadingTrivia: '  ',
 						});
+
+						param
```

Comment:
This can't use the above logic because it `expressionText.includes` can replace the wrong variables.